### ISSUE TITLE
rec_r34_vd_none_bilstm_ctc.yml 转onnx input shape要求是32，不是yml中的48，导致识别结果不对，怎么解决？

### ISSUE THREAD

**Issue by nissansz** [PaddlePaddle/Paddle2ONNX#1048]
**Please fill in the information below so that we can solve the problem quickly, Thanks !**

**Describe the bug**

**Informations (please complete the following information):**
 - Inference engine for deployment:
 - Why convert to onnx：
 - Paddle2ONNX Version: 
 - Email/Wechat/Phone:

**Screenshots**

**Additional context**

rec_r34_vd_none_bilstm_ctc.yml 转onnx input shape要求是32，不是yml中的48，导致识别结果不对，怎么解决？

Train:
  dataset:
    name: SimpleDataSet
    data_dir: F:/
    label_file_list: F:/paddletest.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 48, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 256
    drop_last: True
    num_workers: 8

用--input_shape_dict "{'x':[-1,3,-1,-1]}" 转换得到的模型也报错

--input_shape_dict "{'x':[-1,3,48,-1]}"
failed:Node (p2o.Reshape.0) Op (Reshape) [ShapeInferenceError] Invalid Target shape product of 0

**Comment by yeliang2258** [comment]
https://github.com/PaddlePaddle/Paddle2ONNX/tree/develop/tools/paddle#2-%E4%BF%AE%E6%94%B9-paddle-%E6%A8%A1%E5%9E%8B%E8%BE%93%E5%85%A5-shape
先用这个脚本将Paddle模型的输入修改为你需要的输入，然后再转ONNX

**Comment by nissansz** [comment]
按照上述命令重新得到inference.pdmodel，然后再转onnx，好像还是没显示48，但是已经可以用48运行onnx代码了

![image](https://user-images.githubusercontent.com/41010669/223002774-37ed0c56-edb0-43ec-9b34-050794c9945c.png)

**Comment by nissansz** [comment]
用这个命令可以导出infer模型，inference.pdiparams.info 会更新

py -3 tools/export_model.py -c "F:/rec_r34_vd_none_bilstm_ctc.yml" -o Global.checkpoints="C:/Users/Public/cnjpres34/best_accuracy"  Global.save_inference_dir="C:/Users/Public/cnjpres34/best_accuracy"


用你上面提到的命令，inference.pdiparams.info 不会更新，有办法同时更新inference.pdiparams.info 吗？
python F:\pycharm2020.2\Paddle2ONNX-develop\tools\paddle\paddle_infer_shape.py --model_dir "C:/Users/Public/cnjpres34/best_accuracy" --model_filename inference.pdmodel --params_filename inference.pdiparams --save_dir C:/Users/Public/cnjpres34/best_accuracy --input_shape_dict="{'x':[-1,3,-1,-1]}"
 

好像padding方法不同，
onnx的结果反而更准
 
infer: F:\03.png:('見つけたぞLの一族。', 0.9162090420722961, '見つ{0.59}けたぞL{0.79}の一{0.78}族。')

onnx: [('見つけたぞ Lの一族。', 0.9032566)]
